x-pack/plugins/watcher/server/routes/api/watch/register_save_route.ts

```diff
@@ -21,23 +20,11 @@ const bodySchema = schema.object(
   {
     type: schema.string(),
     isNew: schema.boolean(),
+    isActive: schema.boolean({ defaultValue: true }),
```

Comment:
I'm not sure a default value makes sense here. I think it would make the code clearer if we required the client to define active status, even for defaults. WDYT?

Reply:
I was using this as a guide https://www.elastic.co/guide/en/elasticsearch/reference/current/watcher-api-put-watch.html#watcher-api-put-watch-query-params. I don't feel too strongly about it either way, but if we consider that our endpoint is already a bit different we can require the active flag :+1:

Reply:
Hmm I see. My instinct is to optimize the API for our UI's use cases and not place emphasis on mirroring the ES API. So is the default value useful to the client? Just some thoughts, I think you should make the final call.